lib/sidekiq/cron/schedule_loader.rb

```diff
@@ -0,0 +1,16 @@
+require "sidekiq"
+require "sidekiq/cron/job"
+
+if Sidekiq.server?
+  Sidekiq.configure_server do |config|
+    schedule_file = config.options[:cron_schedule_file] || "config/schedule.yml"
+
+    if File.exist?(schedule_file)
+      config.on(:startup) do
+        if File.exist?(schedule_file)
```

Comment:
do we need this to be checked a couple of times?

Reply:
True we don't, I missed it during refactoring. Have updated it.